daemons/ipa-sam/ipa_sam.c

```diff
@@ -992,6 +994,49 @@ static bool ipasam_uid_to_sid(struct pdb_methods *methods, uid_t uid,
 	return ret;
 }
 
+typedef NTSTATUS (*process_entry_fn)(struct ipasam_private *priv, LDAPMessage *entry, TALLOC_CTX *ctx, void *state);
+static NTSTATUS ipasam_search_entries(struct pdb_methods *methods,
+				      struct ipasam_private *priv,
+				      TALLOC_CTX *ctx,
+				      char *filter,
+				      const char **attrs_list,
+				      process_entry_fn process_entry,
+				      void *state)
+{
+	LDAPMessage *result = NULL;
+	LDAPMessage *entry = NULL;
+	int rc;
+	NTSTATUS res;
+
+	if ((priv == NULL) || (ctx == NULL) || process_entry == NULL) {
```

Comment:
For consistency the final conditional should also be in parens IMHO.

Reply:
Fixed, thanks